### ISSUE TITLE
DPI documentation contains table-images which are inaccessible to screen readers

### ISSUE THREAD

**Issue by andrum99** [raspberrypi/documentation#2458]
The documentation for DPI (at https://www.raspberrypi.com/documentation/computers/raspberry-pi.html#parallel-display-interface-dpi) includes two tables which are images rather than text, and are therefore not accessible to screen readers.

This issue is to judge interest in me fixing this, before I begin work on it.

**Comment by aallan** [comment]
I would be happy to see a PR replacing the images with Asciidoc tables.
